port_inspector/beetle_detection/evaluation_method.py

```
@@ -0,0 +1,242 @@
+# flake8: noqa
```

good idea adding flake to ignore errors, we didnt write this code